frb_codegen/src/generator/dart/ty.rs

```diff
@@ -9,6 +9,12 @@ pub trait TypeDartGeneratorTrait {
         None
     }
 
+    /// Body of `Wire2Api::wire2api` function.
+    ///
+    /// # Safety
+    ///
+    /// `Wire2Api::wire2api` must happen for all fields.
+    /// Early return is unacceptable.
```

Comment:
uber-nit: would be great to explain it a bit more (e.g. paste the case where I got confused previously - since that may be non-intuitive to programmers at first glance)